_docs/reference-field.md

```diff
@@ -79,6 +80,12 @@ Fully featured example:
 The above example configures the Reference Field so that:
 
 - multiple references can be added
-- only File and Folder content types can be added as references
+- only *File* and *Folder* content types can be added as references
 - content can only be referenced from under _/Root/System_ and _/Root/Sites_
-- by default _/Root/System/SystemPlugins_ and _/Root/System/Schema_ is referenced
\ No newline at end of file
+- by default _/Root/System/SystemPlugins_ and _/Root/System/Schema_ is referenced
+
+## Related Links
+- [Field](/docs/field)
+- [ReferenceGrid Field Control]()
+- [Node for developers](/docs/node-for-developers)
+- [ContentPicker]()
```

Comment:
pls remove this part

Reply:
I've removed the parts that you've requested in two new commits. Should I open a new PR?

Reply:
No, you do'nt have to create another pr. Thank you again for your help!